build-n-publish.sh

```diff
@@ -5,7 +5,7 @@ pkgVersion=${1:-"v0.0.0-build0"}
 cdnBaseUrl=${2:-"//cdn.fromdoppler.com/$pkgName"}
 pkgBuild=${4:-0}
 pkgCommitId=${5:-$(git rev-parse HEAD)}
-environments="int qa development production"
+environments="int qa development production local"
```

Comment:
What do you think about using `demo` in place of `local`?

Reply:
Sure, seems fine. Once this works.. 😄